server/lobbyconnection.py

```diff
@@ -454,45 +453,6 @@ def _set_user_agent_and_version(self, user_agent, version):
             metrics.user_agent_version.labels(str(version)).inc()
         self._version = version
 
-    async def _check_version(self):
-        versionDB, updateFile = self.player_service.client_version_info
-        update_msg = {
-            "command": "update",
-            "update": updateFile,
-            "new_version": versionDB
-        }
-
-        if not self.user_agent or "downlords-faf-client" not in self.user_agent:
-            await self.send_warning(
-                "You are using an unofficial client version! "
-                "Some features might not work as expected. "
-                "If you experience any problems please download the latest "
-                "version of the official client from "
-                f'<a href="{config.WWW_URL}">{config.WWW_URL}</a>'
-            )
```

Comment:
I think we do want to keep the user agent check

Reply:
Yes, I agree.

Reply:
OK, I going to fix that.